src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -103,6 +107,7 @@ public MgmtOutputLibrary()
             // these dictionaries are initialized right now and they would not change later
             RawRequestPathToOperationSets = CategorizeOperationGroups();
             ResourceDataSchemaNameToOperationSets = DecorateOperationSets();
+            _schemaToInputEnumMap = new CodeModelConverter(MgmtContext.CodeModel, MgmtContext.Context.SchemaUsageProvider).CreateEnums();
```

Comment:
is this temporary?  If so can you link the follow up item as a comment?

Reply:
Yes, should go away with this one: https://github.com/Azure/autorest.csharp/issues/3770